src/main/java/budgetbuddy/ui/panel/AccountListPanel.java

```diff
@@ -43,9 +43,9 @@ protected void updateItem(Account account, boolean empty) {
                 AccountCard accountCard = new AccountCard(account, getIndex() + 1);
                 setGraphic(accountCard.getRoot());
                 if (account.isActive()) {
-                    accountCard.getRoot().setStyle("-fx-background-color: #005F83");
+                    accountCard.getRoot().setStyle("-fx-background-color: #282547");
                 } else {
-                    accountCard.getRoot().setStyle("-fx-background-color: null");
+                    accountCard.getRoot().setStyle("-fx-background-color: #52667d");
                 }
```

Comment:
This should really be done in CSS..

Reply:
I agree but I couldn't find a way... alternatively I could `setId` instead of specific colour.